src/HashedExpression/Interp.hs

```diff
@@ -12,912 +21,1011 @@ import Data.Map (Map, fromList)
 import qualified Data.Map as Map
 import Debug.Trace (traceId, traceShowId)
 import GHC.TypeLits (KnownNat)
-import HashedExpression.Internal.Expression
-  ( C,
-    ET (..),
-    Expression (..),
-    ExpressionMap,
-    Node (..),
-    NodeID,
-    NodeID,
-    R,
-    Scalar,
-  )
-import HashedExpression.Internal.Node
-import HashedExpression.Internal.Utils
-import HashedExpression.Prettify (prettify, showExp)
-import HashedExpression.Value
+import HashedExpression
+    ( C
+    , ET(..)
+    , Expression(..)
+    , ExpressionMap
+    , Node(..)
+    , One
+    , R
+    , Scalar
+    , Three
+    , Two
+    )
+import HashedNode
+import HashedPrettify (prettify, showExp)
+import HashedUtils
```

Comment:
Same with the imports. I guess you were on another branch where all of these files are not renamed?